trajectory_executor/com_serial.py

```diff
@@ -59,11 +61,13 @@ def sensor_data_request_callback(self, request: SensorDataRequest, response: Sen
 
     # simple deepcopy for dicts with primitives datatypes
     data_packet = json.loads(json.dumps(data_packet_struct))
```

Comment:
ummm are you just trying to deep copy a dict?:

```python
import copy
data_packet = copy.deepcopy(data_packet_struct)
```

Would recommend using `copy` cause it's based off of views, so it has significantly less overhead than a double-JSON conversion.

Reply:
Gotcha. I'll update it to use `copy`